regression/goto-cc-static/chain.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+goto_cc=$1
+cbmc=$2
```

Comment:
The other commit changes this -- some aspect of them probably should be squashed (should this always have been goto-instrument?)